### PR TITLE
Call deluser/delgroup from purge

### LINKED ISSUE #214: deluser/delgroup should be called from purge, not remove
Currently 

`del*` is called for lots of conditions:
https://github.com/wazuh/wazuh-packages/blob/3615935c9c0ad10e4aebb91487a051c4b26a4ff7/debs/SPECS/3.1.0/wazuh-agent/debian/postrm#L7-L14

This is dead wrong and results in broken systems when one uninstalls and then reinstalls `wazuh-agent`.

`del*` should only be called for `purge` in debian systems.

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR closes  #214, I have moved the calls to deluser and delgroup to the purge section of the postrm file.

Regards,
Daniel Folch

### CODE REVIEW

File: debs/SPECS/3.10.0/wazuh-manager/debian/postrm [hunk 1]
```diff
@@ -72,6 +59,18 @@ case "$1" in
     ;;
 
     purge)
+        if getent passwd | grep -q "^ossecr" ; then
```
Comment (BraulioV) [inline debs/SPECS/3.10.0/wazuh-manager/debian/postrm]: This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?

File: debs/SPECS/3.10.0/wazuh-agent/debian/postrm [hunk 2]
```diff
@@ -55,6 +49,12 @@ case "$1" in
     ;;
 
     purge)
+        if getent passwd | grep -q "^ossec" ; then
```
Comment (BraulioV) [inline debs/SPECS/3.10.0/wazuh-agent/debian/postrm]: This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?

Comment (BraulioV) [inline debs/SPECS/3.10.0/wazuh-agent/debian/postrm]: This change is ok but the indentation in some lines are tabs and in other are spaces. Can you change them to spaces?

Comment (DFolchA) [reply debs/SPECS/3.10.0/wazuh-agent/debian/postrm]: Done  in https://github.com/wazuh/wazuh-packages/pull/227/commits/c39e24454ceb2e58427226d03222a53b6d0e91eb

**Pull request merged**